modules/ROOT/pages/usage-metrics.adoc

```diff
@@ -7,14 +7,22 @@ Anypoint Platform captures usage metrics across products, but not all metrics ar
 |===
 |Product |Metric Description |Usage Details
 
-.3+|Mule Runtime	
+.6+|Mule Runtime	
 |Mule flow: A flow within a deployed and running Mule app that contains a Mule event source or route APIKit request. 
 |Flows are aggregated using a Max Concurrent model. The usage for a month is the highest number of flows that exist in a single given hour during a month.
 
 |Mule message: The container of the core information processed by the runtime. 
 |A Mule message counts as a single unit when an event source triggers it. Messages are aggregated using a sum of all messages sent during a month.
 |Data throughput: The total amount of data transferred in and out of the infrastructure that runs Mule where the Mule app is deployed. 
 |Data throughput counts when the deployed application transfers data to execute its business logic, including but not limited to internal operational network traffic for monitoring, logs, and health checks. Data throughput is aggregated as a sum of all bytes during a month. 
+| Cluster: A set of servers that act as a single deployment target.
+| Cluster capacity is determined by your cluster configuration. 
+| CPU Limit: The maximum amount of CPU resources that a worker node can use.
+a| The CPU limit is aggregated by measuring the amount of CPU usage over a specific period of time, such as an hour or a day. This data is collected and aggregated to calculate the total CPU usage for a particular application. 
++
+The CPU Limit configuration of each application is summarized at each environment ID, then at each business group, and then at the root organization ID for preproduction and production environment types separately.
```

Comment:
```suggestion
The CPU limit configuration of each application is summarized at each environment ID, then at each business group, and then at the root organization ID for preproduction and production environment types separately.
```

Reply:
Not sure if lc is correct. Lc on line 21.

Reply:
I'm only capping Limit if I'm referencing the UI (supposedly).

Reply:
If you're referencing the UI, it should also be bold.